src/main/resources/i18n/strings.properties

```
@@ -325,10 +325,16 @@ vaultOptions.masterkey.forgetSavedPasswordBtn=Forget Saved Password
 vaultOptions.masterkey.recoveryKeyExpanation=A recovery key is your only means to restore access to a vault if you lose your password.
 vaultOptions.masterkey.showRecoveryKeyBtn=Display Recovery Key
 vaultOptions.masterkey.recoverPasswordBtn=Recover Password
-## Auto Lock
-vaultOptions.autoLock=Auto-Lock
-vaultOptions.autoLock.lockAfterTimePart1=Lock when idle for
-vaultOptions.autoLock.lockAfterTimePart2=minutes
+## Health
+vaultOptions.health=Vault Health
```

```suggestion
vaultOptions.health=Health Check
```

Since everything is vault-related in "Vault Options", I'd prefer a different name.